WordPress/Classes/ViewRelated/NUX/SiteCreationFields.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+/// Singleton class to contain options selected by the user
+/// during the Site Creation process.
+///
+class SiteCreationFields {
+
+    // MARK: - Properties
+
+    private static var privateShared: SiteCreationFields?
```

Comment:
Since we don't need ObjC support (🎉) I think we can simplify by instantiating the static value immediately. Something like:

`static var privateShared: SiteCreationFields = SiteCreationFields()`

If you want to keep a private var backing the value, it could be accessed via a computed var instead of a method. Let me know what you think :D